dbms/src/Functions/FunctionBinaryArithmetic.h

```diff
@@ -439,7 +468,7 @@ struct BinaryOperationTraits
 };
 
 
-template <template <typename, typename> class Op, typename Name, bool valid_on_default_arguments = true>
+template <template <typename, typename> class Op, typename Name, bool valid_on_default_arguments = true, bool is_bit_operation = false>
```

Comment:
But we can embed `is_bit_operation` directly to Op, it will make the code more clear.

Reply:
fixed (and changed to `allow_fixed_string` to be complete with the `allow_decimal`)